### ISSUE TITLE
Multilable Classification

### ISSUE THREAD

**Issue by sepidehamiri** [sepidehamiri/personal-research#4]
from sklearn.preprocessing import MultiLabelBinarizer

MultiLabelBinarizer().fit_transform(labels)
